mongo/doc.go

```diff
@@ -110,7 +110,7 @@
 // - Go Driver v1.2.0 requires libmongocrypt v1.0.0 or higher
 // - Go Driver v1.5.0 requires libmongocrypt v1.1.0 or higher
 // - Go Driver v1.8.0 requires libmongocrypt v1.3.0 or higher
-// - Go Driver v1.10.0 requires libmongocrypt v1.5.0 or higher
+// - Go Driver v1.10.0 requires libmongocrypt v1.5.2 or higher
```

Comment:
Optional: This is slightly odd messaging and conflates compatibility requirements with libmongocrypt version retraction. My understanding is that libmongocrypt v1.5.0 and v1.5.1 contain serious bugs that can cause data corruption, but are technically build-compatible with Go Driver v1.10.0. Consider adding a note that describes why libmongocrypt v1.5.0/v1.5.1 shouldn't be used.

Reply:
I have updated to say v1.5.0 is required but added a note to say 1.5.2 is required to avoid the severe bug in RewrapManyDataKey.